docs/colang_2/overview.rst

```diff
@@ -89,9 +89,9 @@ You can migrate your Colang 2.0-alpha bots to 2.0-beta using the following comma
 .. code-block:: console
 
     nemoguardrails convert "path/to/2.0-alpha/version/bots" --from-version "2.0-alpha"
-Additionally, you can enable the `--validate` flag to check if the migrated files do not raise any Colang syntax errors:
-.. code-block: console
-    nemoguardrails convert "path/to/2.0-alpha/version/bots" --from-version "2.0-alpha" --validate
+
+Additionally, you can add the ``--validate`` flag to check if the migrated files do not raise any Colang syntax errors.
```

Comment:
Maybe we could add a warning that thiss will migrate the files in-place an overwrite the files. In case someone does not have a backup?